src/args.rs

```diff
@@ -32,48 +29,78 @@ fn parse_presets_path(path: &str) -> anyhow::Result<PresetsPath> {
 #[clap(name = "alma", about = "Arch Linux Mobile Appliance", version, author)]
 pub struct App {
     /// Verbose output
-    #[structopt(short = 'v', long = "verbose")]
+    #[clap(short = 'v', long = "verbose")]
     pub verbose: bool,
 
-    #[structopt(subcommand)]
+    #[clap(subcommand)]
     pub cmd: Command,
 }
 
 #[derive(Parser, Debug, Clone)]
 pub enum Command {
-    #[clap(name = "create", about = "Create a new Arch Linux USB")]
+    #[clap(name = "create", about = "Create a new Arch Linux bootable system")]
     Create(CreateCommand),
-
-    #[clap(name = "chroot", about = "Chroot into exiting Live USB")]
+    #[clap(name = "install", about = "Install this system to another disk")]
+    Install(InstallCommand),
+    #[clap(name = "chroot", about = "Chroot into an existing ALMA system")]
     Chroot(ChrootCommand),
-
-    #[clap(name = "qemu", about = "Boot the USB with Qemu")]
+    #[clap(name = "qemu", about = "Boot the ALMA system with Qemu")]
     Qemu(QemuCommand),
 }
 
+#[derive(ValueEnum, Debug, Clone, Copy, Serialize, Deserialize, PartialEq, Eq, Default)]
+#[serde(rename_all = "lowercase")]
+pub enum SystemVariant {
+    #[default]
+    Arch,
+    Omarchy,
+}
+
+impl fmt::Display for SystemVariant {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        write!(
+            f,
+            "{}",
+            match self {
+                SystemVariant::Arch => "arch",
+                SystemVariant::Omarchy => "omarchy",
+            }
+        )
+    }
+}
+
+#[derive(ValueEnum, Debug, Clone, Copy, Serialize, Deserialize, PartialEq, Eq, Default)]
+#[serde(rename_all = "lowercase")]
+pub enum FilesystemTypeArg {
+    #[default]
+    Ext4,
+    Btrfs,
+    Vfat,
```

Comment:
vfat isn't really an option for clap, no?

Reply:
Good catch! I simplified it too much before thanks!